AUTHORS

```diff
@@ -2316,5 +2315,374 @@ Ben Newman <ben@meteor.com>
+Robin Drexler <robin.drexler@xing.com>
+ZauberNerd <zaubernerd@zaubernerd.de>
+G. Carcaci <giuseppecarcaci@gmail.com>
+Jackson Chui <14085209+haiXchuus@users.noreply.github.com>
+Marc Posth <marc.posth@shopify.com>
+Simona Cotin <simona.cotin@gmail.com>
+Mathieu Pavageau <mth.pvg@gmail.com>
+Clement <clement.sch@free.fr>
+razvanbh <razvy.rotaru@gmail.com>
+reineke-fox <nils@reineke.io>
+Mark Arranz <mark.arranz@gmail.com>
+alyssaq <alyssaquek@gmail.com>
+Manish Poddar <manish.poddar@epoch.com>
+szabolcsit <szabolcsit@hu.ibm.com>
+Kevin Seidel <kevin.seidel@karrieretutor.de>
+Jonah Polack <jonah.polack@gmail.com>
+Paul Isache <paul@isache.com>
+Artur Daschevici <artur.daschevici@nearform.com>
+Grant Carthew <grant@carthew.net>
+Alex Seifert <alex@alexseifert.com>
+Amer Alimanović <x@amer8.com>
+Nikita Malyschkin <nikita.malyschkin@gmail.com>
+ivan.filenko <filenko@uchi.ru>
+Marcus Scott <mmmrscott@gmail.com>
+Berry de Witte <bdwitte@wehkamp.nl>
+saurabhSiddhu <sourabh.siddhu@gmail.com>
+Marie Terrier <marie.terrier1@gmail.com>
+John Mc Quillan <john.mcq1llan@gmail.com>
+Ulises Santana Suárez <ulisesantana@gmail.com>
+Yehiyam Livneh <yehiyam@gmail.com>
+Aivo Paas <aivo.paas@pipedrive.com>
+James Herrington <james@getimpala.com>
+Natalie Cluer <natalie@getimpala.com>
+Dmitry Igrishin <dmitigr@gmail.com>
+Martin Kask <martin.kask@pipedrive.com>
+Alessandro Gatti <ale.gatti96@gmail.com>
+Vladyslav Kopylash <vladik.kopilash@gmail.com>
+Osmond van Hemert <osmond@vanhemert.nu>
+msmichellegar <msmichellegar@gmail.com>
+Robert Pamely <info@robertpamely.com>
+Lauri Piisang <lauri.piisang@pipedrive.com>
+kiyomizumia <melissa@stroly.jp>
+Paul Hodgson <prhodgson1@gmail.com>
+Emanuel Kluge <kraemerseele@gmx.de>
+Andrey Melikhov <amel.truelife@gmail.com>
+Philipp Dunkel <pip@pipobscure.com>
+Steven Gabarro <bewchabbacc@gmail.com>
+Anatoli Papirovski <anatoli.papirovski@postmates.com>
+Dan Corman <dan.corman@gmail.com>
+Rodger Combs <rodger.combs@gmail.com>
+Charalampos Fanoulis <38255093+cfanoulis@users.noreply.github.com>
+Julia <jkmcgeoghan@myseneca.ca>
+Kanika Singhal <singhalkanika198@gmail.com>
+Gagandeep Singh <codegagan@gmail.com>
+Lakshmi Shanmugam <lshanmug@in.ibm.com>
+Jayasankar <jayasankar.m@gmail.com>
+Pushkal B <pushkalb123@gmail.com>
+potham <pothamk@gmail.com>
+Shubham Urkade <shubhamurkade1@gmail.com>
+Nikhil M <m_nikhil@outlook.com>
+leeight <leeight@gmail.com>
+Selvaraj <selvarajrch@gmail.com>
+Mayank Asthana <mayankasthana1993@gmail.com>
+Simon Bruce <simon@blacksun.cx>
+NoSkillGirl <singhpooja240393@gmail.com>
+apoorvanand <apoorwanand@gmail.com>
+Jay Arthanareeswaran <jarthana@in.ibm.com>
+Amanpreet <amanpreet6309108@gmail.com>
+kanishk30 <kanishk71@gmail.com>
+UjjwalUpadhyay <ujjwalkool1990@gmail.com>
+Namit Bhalla <oyenamit@gmail.com>
+Matteo <teorossi82@yahoo.it>
+dexterleng <thequicklab@gmail.com>
+Kunda Sunil Kumar <kunda.sunilkumar@gmail.com>
+rahulshuklab4u <rahulshuklab4u@gmail.com>
+gengjiawen <technicalcute@gmail.com>
```

Comment:
Back then I am young and simple, now I think my email address is silly 😭

Reply:
@gengjiawen You can always change it for new commits with `git config --global user.email foo@bar.com`, and you can always choose one for the `AUTHORS` file if you want? :)

(Also, it’s not silly. :slightly_smiling_face:)

Reply:
Thanks. Maybe later :)